components/Header/index.js

```diff
@@ -1,15 +1,25 @@
-import Image from "next/image";
 import React from "react";
 import Link from "next/link";
+import styled from "styled-components";
 
-const Header = () => {
+const StyledHeader = styled.header`
+  background-color: #987554;
+  margin-top: 0;
```

Comment:
I think you could remove this because the `<header>` doesn't have some kind of default margin.

```suggestion
```